src/index.html

```diff
@@ -215,7 +217,7 @@ <h1>COVID-19 Spread Simulator 🦠</h1>
   </main>
   
   <footer>
-    Based on <a target="_blank" rel="noopener nofollow" href='https://www.washingtonpost.com/graphics/2020/world/corona-simulator/'>Washington Post article</a> | <a href='https://github.com/noiapp/covid-19-spread-simulator' target="_blank" rel="noopener nofollow">Source Code</a> | By <a href='https://midu.dev'>@midudev</a>
+    Based on <a target="_blank" rel="noopener nofollow" href='https://www.washingtonpost.com/graphics/2020/world/corona-simulator/'>Washington Post article</a> | <a href='https://github.com/midudev/covid-19-spread-simulator' target="_blank" rel="noopener nofollow">Source Code</a> | By <a href='https://midu.dev'>@midudev</a> | Extended by <a href='https://www.linkedin.com/in/marco-pierobon-a82ab646/'>Marco Pierobon</a>
```

Comment:
Consider to move attribution to a new `CONTRIBUTORS.md` file, where you can store your info alongside a summary of the changes you did. IMHO the right way would be a link to the CONTRIBUTOR file.

Reply:
Removed - we'll discuss attribution later, not a priority